deps/cares/src/ares_init.c

```diff
@@ -1000,11 +1025,6 @@ static int get_DNS_AdaptersAddresses(char **outptr)
       }
       else if (namesrvr.sa->sa_family == AF_INET6)
       {
-        /* Windows apparently always reports some IPv6 DNS servers that
-         * prefixed with fec0:0:0:ffff. These ususally do not point to
-         * working DNS servers, so we ignore them. */
-        if (strncmp(txtaddr, "fec0:0:0:ffff:", 14) == 0)
-          continue;
```

Comment:
@saghul I think it should be enough to just float the patch actually instead of squashing it into update commit.